components/item.js

```diff
@@ -0,0 +1,73 @@
+var _createClass = function () { function defineProperties(target, props) { for (var i = 0; i < props.length; i++) { var descriptor = props[i]; descriptor.enumerable = descriptor.enumerable || false; descriptor.configurable = true; if ("value" in descriptor) descriptor.writable = true; Object.defineProperty(target, descriptor.key, descriptor); } } return function (Constructor, protoProps, staticProps) { if (protoProps) defineProperties(Constructor.prototype, protoProps); if (staticProps) defineProperties(Constructor, staticProps); return Constructor; }; }();
+
+function _classCallCheck(instance, Constructor) { if (!(instance instanceof Constructor)) { throw new TypeError("Cannot call a class as a function"); } }
```

Comment:
There are established standards regarding line length that you **must** adopt.  This is not readable code.

Historically  folks enforced an 80 character line length.  Most language communities still hold to this standard with a few -- not JS -- going to 100 or 120.

Stick to 80 characters per line.  If you're uncertain why the standard, you can find really thoughtful pieces on this with a simple web search.

Reply:
These comments appear to be on transpiled files. Not sure why they ended up here instead of in /dist, probably artifacts from an earlier webpack error I was having.

I have deleted the duplicate transpiled files, so they should not be there in my next PR.

The only code I see in my non-transpiled code which is longer than 80 characters that I see off-hand is in my testing, which is my next PR. I will go in and edit those for readability before submitting that PR.

Thank you!